README.md

```diff
@@ -1,10 +1,89 @@
-# calico-material
+# 📌 Calico Material  
 
-[Material](https://m3.material.io/) components for [Calico](https://armanbilge.github.io/calico).
+A collection of **Material Design components** for [Calico](https://armanbilge.github.io/calico/), a Scala.js UI library.  
+This library allows developers to use Google's **Material Design** components in Calico applications.  
 
-### Demo
+---
+
+## 📦 Installation  
+
+Add the following dependency to your **`build.sbt`** file:  
+
+```scala
+libraryDependencies += "com.armanbilge" %%% "calico-material" % "0.1.0"
+```
+
+# 🚀 Getting Started
+To use a Material button in your Scala.js project:
+
+```
+import calico.material.MaterialButton
+import cats.effect.IO
+import com.raquo.laminar.api.L._
+import calico.html.io._
```

Comment:
Laminar is an entirely different UI library from Calico. They are not intended to be used together.